assignment/assignment1/client/src/api/usersAPI/registerUser.ts

```diff
@@ -0,0 +1,28 @@
+import { API_URL } from "../config";
+
+export async function registerUser(username: String, password: String) {
+    try {
+        const response = await fetch(`${API_URL}/auth/register`, {
+            method: 'POST',
+            // body: formData,
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                username,
+                password,
+            }),
+        });
+        if (!response.ok) {
+            throw new Error(`${await response.json()}`)
+        }
+        // if (response.ok) {
+        //     return await response.json(); 
+        // } else {
+        //     throw new Error(`${response}`);
+        // }
```

Comment:
remove commented code

Reply:
i will remove this in the question history branch to prevent extra merge conflicts